spec/controllers/sign_in_controller_spec.rb

```diff
@@ -634,6 +634,87 @@
     end
   end
 
+  describe 'POST logout' do
+    subject { post(:logout, params: {}.merge(refresh_token_param).merge(anti_csrf_token_param)) }
+
+    let(:refresh_token_param) { { refresh_token: refresh_token } }
+    let(:refresh_token) { 'example-refresh-token' }
+    let(:anti_csrf_token_param) { { anti_csrf_token: anti_csrf_token } }
+    let(:anti_csrf_token) { 'example-anti-csrf-token' }
+    let(:enable_anti_csrf) { false }
+
+    before { allow(Settings.sign_in).to receive(:enable_anti_csrf).and_return(enable_anti_csrf) }
+
+    context 'when refresh_token is a random string' do
```

Comment:
Can you add specs for the case when `enable_anti_csrf` is set to true? Luckily it should prob be basically a straight copy from the `refresh` route

Reply:
👍